Foundation/NSThread.swift

```diff
@@ -57,18 +57,30 @@ private func NSThreadStart(_ context: UnsafeMutableRawPointer?) -> UnsafeMutable
 }
 
 open class Thread : NSObject {
-    
+    static internal let _mainThread = Thread(thread: pthread_self())
```

Comment:
perhaps for safety sake we could call this in __CFInitializeSwift to ensure the proper initialization? however that does make the assumption the library is loaded on the main thread (Android can break this rule)